go.mod

```diff
@@ -183,9 +183,10 @@ require (
 )
 
 replace (
+	k8s.io/client-go => k8s.io/client-go v0.25.2
```

Comment:
why hard replace here?

Reply:
Moving to  the 0.26.x go client conflicts with the hypershift api imports due to package changes in 0.26. I pinned to 0.25 to avoid going down the path of sorting through that.

```
github.com/openshift/hypershift/api imports
	k8s.io/client-go/kubernetes/scheme imports
	k8s.io/api/auditregistration/v1alpha1: module k8s.io/api@latest found (v0.26.1), but does not contain package k8s.io/api/auditregistration/v1alpha1
github.com/openshift/hypershift/api imports
	k8s.io/client-go/kubernetes/scheme imports
	k8s.io/api/batch/v2alpha1: module k8s.io/api@latest found (v0.26.1), but does not contain package k8s.io/api/batch/v2alpha1
github.com/openshift/hypershift/api imports
	k8s.io/client-go/kubernetes/scheme imports
	k8s.io/api/settings/v1alpha1: module k8s.io/api@latest found (v0.26.1), but does not contain package k8s.io/api/settings/v1alpha1
github.com/openshift/hypershift/api/fixtures imports
	sigs.k8s.io/controller-runtime/pkg/client imports
	k8s.io/client-go/metadata: package k8s.io/client-go/metadata provided by k8s.io/client-go at latest version v0.26.1 but not at required version v12.0.0+incompatible
github.com/openshift/hypershift/availability-prober imports
	k8s.io/client-go/discovery imports
	github.com/googleapis/gnostic/OpenAPIv2: module github.com/googleapis/gnostic@latest found (v0.6.9), but does not contain package github.com/googleapis/gnostic/OpenAPIv2
github.com/openshift/hypershift/control-plane-operator/controllers/awsprivatelink imports
	k8s.io/client-go/util/workqueue imports
	k8s.io/apimachinery/pkg/util/clock: module k8s.io/apimachinery@latest found (v0.26.1), but does not contain package k8s.io/apimachinery/pkg/util/clock
```